custom/icds_reports/static/js/directives/adolescent-girls/adolescent-girls.directive.js

```diff
@@ -58,7 +58,7 @@ function AdolescentWomenController($scope, $routeParams, $location, $filter, dem
     };
 
     vm.chartOptions = vm.getChartOptions(options);
-    vm.chartOptions.chart.width = 1100;
+    vm.chartOptions.chart.width = isMobile ? '' : 1100;
```

Comment:
if the width is empty it just takes the available space?

Reply:
also is there a reason not to define this inside `baseControllers.getChartOptions` since it seems to always be the same in the subclasses?

Reply:
Yes, if it is empty, it takes available space. 
I thought to define it in basecontrollers, but only certain charts had this width defined.. In other charts.. width was undefined i.e empty. So I left it as is

Reply:
ok, makes sense, thanks